### DISCUSSION TITLE
Translating a model causes an issues with blocks component

### DISCUSSION THREAD

**Discussion by mikemoooo** [payloadcms/payload#1145] [Q&A]
# Bug Report

When updating the content of a page in another locale, I have had the issue that within a blocks container an empty object is added. This causes the layout blocks to start counting at 2 and prevents adding new block. When trying to save the console shows the error:
Uncaught TypeError: Cannot destructure property
'blockType' of as it is undefined.

The screenshot below shows the initial state after switching to the new locale.
The issue can be fixed by updating the mongodb and removing the empty object.

## Expected Behavior

When no content is translated for the block component, the component should be empty.

## Detailed Description

Payload version: 1.0.30

![SCR-20220901-qxe](https://user-images.githubusercontent.com/30129463/187975161-813c39e5-63e9-4a06-a09a-02bb6c06afa1.png)
![SCR-20220901-qxv](https://user-images.githubusercontent.com/30129463/187975220-43bde907-0a14-4a65-aaac-e2d43566c937.png)

**Comment by denolfe** [comment]
Hey @mikemoooo, I attempted to recreate your issue, but I was unable to. Could you provide some more detail?

Here are the steps I performed:

- On a collection with a localized block, Create new
- Populate block for `en`
- Save
- Switch to `es`
- Save without populating

Are these similar to the steps you performed to get this issue? If you can provide a slimmed down version of your collection config to recreate, that would also be helpful.

**Comment by jmikrut** [comment]
@mikemoooo just checking in here for any possible reproduction steps as we can't reproduce the issue. I will convert this to a Discussion for now, and if we can replicate the issue, we will re-open accordingly.

Thank you!
